screwdriver/store/store.go

```diff
@@ -35,17 +40,92 @@ type Command struct {
 	Meta *api.Command
 }
 
+func (e ResponseError) Error() string {
+	return fmt.Sprintf("Store API %d %s", e.StatusCode, e.Reason)
+}
+
+func commandURL(meta *api.Command) (string, error) {
+	if meta.Format == "binary" {
```

Comment:
I think `meta` makes us a little confusing. We already have `meta-cli` and metadata. Why isn't this `command`?

Reply:
Store object also has a Command object and the Command object has an api.Command. I thought use `command` as an api.Command is complicated. So I used `meta` as an abbreviation of meta-information. 

I did not think about meta-cli. I will think better name of api.Command variable.

Reply:
I missed Command of Store.  Store is used with Binary type so how about `BinaryCommand`?
`meta` is a common word but I think we should not use in this case.
I cannot show any better word...

Reply:
When I look at [command design (Details.Formats.binary)](https://github.com/screwdriver-cd/screwdriver/blob/master/design/commands.md), we will use /commands endpoint to get Command from Store. I think it is better to declare same name of endpoint.

Reply:
How about use `spec` for api.Command?
Also store.Command struct has `Spec` as an api.Command.

Reply:
Thanks then I change meta to spec